book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [ ] Getters and setters for rentals should be present, please do so to comply with the project requirements.

Reply:
Thank you for your valuable comment. I changed it as you can see in the image below:
![attr2](https://user-images.githubusercontent.com/60292761/141265572-58fd98c6-4414-495e-88d2-760c589df339.PNG)